nosqlbench/http-jsonapi-crud-basic.md

```diff
@@ -58,6 +58,7 @@ Note that error handling is set to `errors=timer,warn`, which means that in case
 ## Workload Parameters
 
 - `docscount` - the number of documents to process in each step of a scenario (default: `10_000_000`)
+- `connections` - number of HTTP2 connections to be shared between the threads (default: `20`) 
```

Comment:
how does this relate to the threads? If I set threads=10 and connections=20 what happens? Should we provide guidance on whether these need to be tuned together?

Would it make sense to reframe this as a number of connections per thread, or perhaps I misunderstand how this works?

Reply:
* `threads=10 and connections=20` - each thread own connection..
* connections per thread would be great, but I don't know how to do that? do you know?

Reply:
Let's not worry about this now.